File: Domain/Aggregates/InventoryAggregate/Item.cs
namespace Domain.InventoryAggregate;

public class Item
{
    
    public string Name { get; }

    public DateOnly ExpirationDate { get; }
    
    public ItemType Type { get; }

    public Item(string name, DateOnly expirationDate, ItemType type)
    {
        Name = name;
        ExpirationDate = expirationDate;
        Type = type;
    }

}

File: src/Front/ui/src/model/Item.ts
export class Item {
  name: string;
  type: string;
  expirationDate: string;

  constructor(name: string, type: string, expirationDate: string) {
    this.name = name;
    this.type = type;
    this.expirationDate = expirationDate;
  }
}
